<?xml version="1.0"?>
<launch>
    <rosparam file="$(find insta360_ros_driver)/config/intrinsics.yaml" command="load"/>
    <param name="bag_dir" value="$(find insta360_ros_driver)/bag"/>
    <param name="undistort" value="true"/>

    <node name="undistortion_node" pkg="insta360_ros_driver" type="undistortion.py" output="screen"/>
</launch>